e2e-tests/portal/cypress/e2e/journeys/dashboard/dashboard-deals/dashboard-deals-filter-created-by-you.spec.js

```diff
@@ -61,16 +61,22 @@ context('Dashboard Deals filters - Created by you', () => {
 
   before(() => {
     cy.login(BANK1_MAKER1);
+  });
+
+  beforeEach(() => {
+    cy.saveSession();
+
     dashboardDeals.visit();
+    filters.showHideButton().click();
+  });
+
+  it('Visit the correct page', () => {
```

Comment:
Not a blocker, could keep in line with the other tests `it('should redirect to the correct page'`

Reply:
Renamed.